src/dashboard/src/components/charts/allOrgDonut/updateData.ts

```diff
@@ -0,0 +1,42 @@
+import { convertToStorageSize } from '@/utils';
+import { IStats } from './IStats';
+
+export const updateData = (
```

Comment:
If this is reusable for other chart components we'll extract to a better location.